Formula/proselint.rb

```diff
@@ -0,0 +1,33 @@
+class Proselint < Formula
+  include Language::Python::Virtualenv
+
+  desc "Linter for prose"
+  homepage "http://proselint.com"
+  url "https://files.pythonhosted.org/packages/e6/78/c46f694a0e43ce47d0d6ae089f750131544018dc1119fc7da58ffd4b1e03/proselint-0.7.0.tar.gz"
+  sha256 "094d808d44bf1a60dcb1465749be5cc44f4f6c146c04bc5f28976a833786e830"
```

Comment:
thanks. it's just so livecheck can find the new versions :) alternatively we could use the github tarball for the `url` field but people have a tendency to come along and replace that with the pypi links lol